appveyor.bat

```diff
@@ -1,392 +1,490 @@
-call :downloadfile %GETTEXT64_URL% downloads\gettext64.zip
-7z.exe e -y downloads\gettext64.zip -o%DEPENDENCIES%\gettext64 > nul || exit 1
+@rem Install libintl.dll and iconv.dll
+call :downloadfile "%GETTEXT_32_URL%" downloads\gettext32.zip
+7z.exe e -y downloads\gettext32.zip ^
+  -o%DEPENDENCIES%\gettext32 > nul || exit 1
+
+call :downloadfile "%GETTEXT_64_URL%" downloads\gettext64.zip
+7z.exe e -y downloads\gettext64.zip ^
+  -o%DEPENDENCIES%\gettext64 > nul || exit 1
 
-:: Install winpty
-call :downloadfile %WINPTY_URL% downloads\winpty.zip
-7z.exe x -y downloads\winpty.zip -o%DEPENDENCIES%\winpty > nul || exit 1
+@rem Install winpty
+call :downloadfile "%WINPTY_URL%" downloads\winpty.zip
+7z.exe x -y downloads\winpty.zip -o%WINPTY_DIR% > nul || exit 1
 if /I "%ARCH%"=="x64" (
-	copy /Y %DEPENDENCIES%\winpty\x64_xp\bin\winpty.dll        vim\src\winpty64.dll
-	copy /Y %DEPENDENCIES%\winpty\x64_xp\bin\winpty-agent.exe  vim\src\
+  mklink /H vim\src\winpty64.dll %WINPTY_DIR%\x64\bin\winpty.dll
+  mklink /H vim\src\winpty-agent.exe %WINPTY_DIR%\x64\bin\winpty-agent.exe
 ) else (
-	copy /Y %DEPENDENCIES%\winpty\ia32_xp\bin\winpty.dll       vim\src\winpty32.dll
-	copy /Y %DEPENDENCIES%\winpty\ia32_xp\bin\winpty-agent.exe vim\src\
+  mklink /H vim\src\winpty32.dll %WINPTY_DIR%\ia32\bin\winpty.dll
+  mklink /H vim\src\winpty-agent.exe %WINPTY_DIR%\ia32\bin\winpty-agent.exe
 )
 
-:: Install UPX
-::
-:: This is disabled, because UPX is known to be flagged as
-:: supsicious by AV vendors
-::
-::call :downloadfile %UPX_URL% downloads\upx.zip
-::7z.exe e -y downloads\upx.zip *\upx.exe -ovim\nsis > nul || exit 1
-
-:: Install ShellExecAsUser
-call :downloadfile %SHELLEXECASUSER_URL% downloads\shellexecasuser.zip
-7z.exe x -y downloads\shellexecasuser.zip -o%DEPENDENCIES%\shellexecasuser > nul || exit 1
-copy /Y %DEPENDENCIES%\shellexecasuser\unicode\ShellExecAsUser.dll "%ProgramFiles(x86)%\NSIS\Plugins\x86-unicode"
-
-:: Install Libsodium
-call :downloadfile %LIBSODIUM_URL% downloads\libsodium.zip
+@rem Install UPX
+@rem This is disabled, because UPX is known to be flagged as
+@rem supsicious by AV vendors
+rem call :downloadfile %UPX_URL% downloads\upx.zip
+rem 7z.exe e -y downloads\upx.zip *\upx.exe -ovim\nsis > nul || exit 1
+
+@rem Install ShellExecAsUser
+call :downloadfile "%SHELLEXECASUSER_URL%" downloads\shellexecasuser.zip
+7z.exe x -y downloads\shellexecasuser.zip ^
+  -o%DEPENDENCIES%\shellexecasuser > nul || exit 1
+mklink /H "%ProgramFiles(x86)%\NSIS\Plugins\x86-unicode\ShellExecAsUser.dll" ^
+  %DEPENDENCIES%\shellexecasuser\unicode\ShellExecAsUser.dll
+
+@rem Install Libsodium
+call :downloadfile "%LIBSODIUM_URL%" downloads\libsodium.zip
 7z.exe x -y downloads\libsodium.zip -o%DEPENDENCIES%\ > nul || exit 1
 if /I "%ARCH%"=="x64" (
-	copy /Y %DEPENDENCIES%\libsodium\x64\Release\v143\dynamic\libsodium.dll        vim\src\libsodium.dll
+  mklink /H vim\src\libsodium.dll ^
+  %SODIUM_DIR%\x64\Release\v143\dynamic\libsodium.dll
```

Comment:
```suggestion
  mklink /H vim\src\libsodium.dll ^
    %SODIUM_DIR%\x64\Release\v143\dynamic\libsodium.dll
```
Could you indent all the continued lines (as long as it doesn't change the behavior)?

Reply:
It's done, Takata‐san